### DISCUSSION TITLE
Onyx Nova Pro (eBook) is not supported!?

### DISCUSSION THREAD

**Discussion by paultarasov** [AppGround-io/bluetooth-keyboard-and-mouse-support#165] [Bugs & Problems]
Hi!

I've this device (https://onyxboox.com/boox_novapro) with Android 6.0 and BT 4.1.
When I open App I get this:
"Not supported
This device does not have the required software support to be used as a Bluetooth Keyboard.
Please try again with a different Android device"

What am I doing wrong? What software do I need to install?

Regards,
Paul

**Comment by parkerlreed** [comment]
The requirements for the *receiver* are Android 4.4

For the actual *server* device you need Android 9 or higher (Definitely not 6)

**Reply by paultarasov** [reply]
I want to use this device (Nova Pro) as a keyboard for my PC/IPad. So It must work, bu it doesn't :(

**Reply by parkerlreed** [reply]
The OS on your Nova Pro is too old. Unless they provide an update or someone has created a custom ROM for your device, there's no way to get that on a newer version.

**Reply by paultarasov** [reply]
Ок. thank you for reply.

**Comment by AppGround-io** [comment]
You can try the Android 9.0 update for your device and see if this work with the app.

https://goodereader.com/blog/onyx-boox/onyx-boox-nova-pro-and-note-pro-have-an-android-9-update
